src/components/PokemonCard/PokemonCard.js

```diff
@@ -6,11 +6,11 @@ const PokemonCard = (props) => {
     return (
         <section className="pokemon-card">
             <h2>{ props.name }</h2>
-            <img src={ props.image } className="pokemon-i" alt={`Pokemon ${ props.name }`} />
+            <img src={ props.image } className="pokemon-image" alt={`Pokemon ${ props.name }`} />
```

Comment:
I did image before but I got an alert message in the console saying that it is redundant to use image in a image tag, the readers would read twice, that's the reason why it is -i